application/classes/Ushahidi/Repository/User.php

```diff
@@ -133,6 +138,9 @@ public function isUniqueEmail($email)
 	// RegisterRepository
 	public function register(Entity $entity)
 	{
+
+		$this->updateIntercomUserCount(1);
```

Comment:
I would definitely agree with that. The issue asked for user count though but it would be vastly easier to push the user itself and then intercom filter by deployment to get the count.